Apps/phcybereason/cybereason.json

```diff
@@ -0,0 +1,1497 @@
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string",
+                    "column_name": "data"
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "query processes",
+            "identifier": "query_processes",
+            "description": "Queries a given malop to retrieve all processes",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "malop_id": {
+                    "description": "The ID of the malop for which we want to get process details",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "cybereason malop id"
+                    ],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.malop_id",
+                    "data_type": "string",
+                    "contains": [
+                        "cybereason malop id"
+                    ],
+                    "column_name": "malop_id"
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 1,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.process_id",
+                    "data_type": "string",
+                    "contains": [
+                        "cybereason process id"
+                    ],
+                    "column_name": "process_id",
```

Comment:
Change `column_name`s to have spaces instead of underscores. Example for this: `Process ID`